src/modules/orders/table/row.tsx

```diff
@@ -0,0 +1,123 @@
+import { useTableMenu } from '@/utils/hooks/use-table-menu'
+
+import { OrderStatusChip } from '../StatusChip'
+import { OrderDto } from '../types'
+
+interface ITableRowProps {
+  data: OrderDto
+  onDeleteClick: () => void
+  isDeleteOptionEnabled: boolean
+  onAcceptOrderClick: (id: number) => void
+  onRejectOrderClick: (id: number) => void
+  onCloseOrderClick: (id: number) => void
+}
+
+export const OrdersTableRow = ({
+  data,
+  onDeleteClick,
+  isDeleteOptionEnabled,
+  onAcceptOrderClick,
+  onRejectOrderClick,
+  onCloseOrderClick
+}: ITableRowProps) => {
+  const router = useRouter()
+  const { t } = useTranslation()
+
+  const {
+    menuAnchorEl,
+    isMenuOpen,
+    handleMenuClick,
+    handleMenuClose,
+    handleMenuAction,
+  } = useTableMenu()
+
+  const { id, player, status, scout, createdAt, description, _count: count, match } = data
+
+  return (
+    <StyledTableRow
+      hover
+      key={id}
+      onClick={isMenuOpen ? undefined : () => router.push(`/orders/${id}`)}
+    >
+      <StyledTableCell padding="checkbox">
+        <TableMenu
+          menuAnchorEl={menuAnchorEl}
+          isMenuOpen={isMenuOpen}
+          onMenuClick={handleMenuClick}
+          onMenuClose={handleMenuClose}
+        >
+          {status === 'OPEN' ?
+            <TableMenuItem
+              icon={<AcceptIcon fontSize="small" />}
+              text={t('orders:ACCEPT')}
+              onClick={() => {
+                handleMenuAction(() => onAcceptOrderClick(id))
+              }}
+            />
+            :
+            <>
+              <TableMenuItem
+                icon={<RejectIcon fontSize="small" />}
+                text={t('REJECT')}
+                onClick={() => {
+                  handleMenuAction(() => onRejectOrderClick(id))
+                }}
+              />
+              <TableMenuItem
+                icon={<CloseIcon fontSize="small" />}
+                text={t('CLOSE')}
+                onClick={() => {
+                  handleMenuAction(() => onCloseOrderClick(id))
+                }}
+              />
+            </>}
+          <TableMenuItem
+            icon={<DeleteIcon fontSize="small" />}
+            text={t('DELETE')}
+            onClick={() => {
+              handleMenuAction(onDeleteClick)
+            }}
+            disabled={!isDeleteOptionEnabled}
+          />
+        </TableMenu>
+      </StyledTableCell>
+      <CellWithLink href={getSinglePlayerRoute(player?.slug || '')} label={player ? `${player?.firstName} ${player?.lastName}` : ''} />
+      <StyledTableCell>{player?.primaryPosition.name}</StyledTableCell>
+      <CellWithLink href={getSingleTeamRoute(player?.teams[0].team.slug || '')} label={player?.teams[0].team.name || ''} />
+      <CellWithLink href={getSingleMatchRoute(match?.id || 0)} label={match ? getMatchDisplayName({ awayTeamName: match.awayTeam.name, homeTeamName: match.homeTeam.name, competitionName: match.competition.name }) : ''} />
```

Comment:
```suggestion
        {match ? (
          <CellWithLink
            href={getSingleMatchRoute(match.id)}
            label={getMatchDisplayName({
              homeTeamName: match.homeTeam.name,
              awayTeamName: match.awayTeam.name,
            })}
          />
        ) : (
          <StyledTableCell>-</StyledTableCell>
        )}
```

Same here. And please omit `competitionName` in table cells, it's too much text